index.html

```diff
@@ -63,5 +63,43 @@ <h1>
     
         </div>
     </section>
+
+   <!-- ------------------------------------mobile section-------------------- -->
+<section class="section-nav">
+
+   <header class="header-mobile">
+    <nav>
+        <img src="images/section_header.png" alt="logo">
+    </nav>
+   </header>
+
+<section class="section-mobile">
+   <div class="project-title">
+    <h1>Tonic</h1>
+    <ul class="top-list">
+        <li>CANOPY</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+    </ul>
+   </div>
+    <p>
+        A daily selection of privately personalized 
+        reads; no accounts or 
+        sign-ups required.
+    </p>
+
+    <ul class="list-bottom">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <div class="action">
+      <button>See Project</button>
+    </div>
+    
+
+</section>
+</section>
```

Comment:
- [ ] According to your Figma template, you need to enclose this "work-card" in an edge-rounded container, to highlight the card content from the background color:

Your design:
![image](https://user-images.githubusercontent.com/52635378/170833309-4a5cd2fc-16f3-4c6e-99ca-168caa1179d7.png)
Figma:
![image](https://user-images.githubusercontent.com/52635378/170833327-9ffaec0b-51e1-4061-a604-50f93785e4e6.png)